src/hci/gnomex/security/ActiveDirectory.java

```diff
@@ -39,122 +38,123 @@
 import javax.naming.directory.SearchControls;
 import javax.naming.directory.SearchResult;
 
-import org.dom4j.Attribute;
+
 
 /**
  * Query Active Directory using Java
- * 
+ *
  * @filename ActiveDirectory.java
  * @author <a href="mailto:jeeva@myjeeva.com">Jeevanandam Madanagopal</a>
  * @copyright &copy; 2010-2012 www.myjeeva.com
  */
 public class ActiveDirectory {
-	// Logger
-	private static final Logger LOG = Logger.getLogger(ActiveDirectory.class.getName());
+    // Logger
+    //private static final Logger LOG = Logger.getLogger(ActiveDirectory.class.getName());

```

Comment:
Why explicit class reference over the import?  Also, the accepted java naming convention for constants is all caps.  Makes it easier to identify constants in your code.